packages/fast-usdc/src/clientSupport.js

```diff
@@ -0,0 +1,110 @@
+// @ts-check
+import { AmountMath } from '@agoric/ertp';
+import { assertAllDefined } from '@agoric/internal';
+
+// Reusable amount scaling similar to inter-protocol's approach
+const COSMOS_UNIT = 1_000_000n;
+const scaleDecimals = num => BigInt(num * Number(COSMOS_UNIT));
+
+/**
+ * @param {Pick<
+ *   import('@agoric/vats/tools/board-utils.js').AgoricNamesRemotes,
```

Comment:
Let's please stop using `BoardRemote` and hence `AgoricNamesRemotes`. Just use the ordinary types like `Brand`.

ref
 - #10491

Reply:
I agree eventually but can you accept that as being out of scope? The `inter-protocol` clientSupport uses `AgoricNamesRemotes` and I think if we're going to change it we should do both.

Reply:
> if we're going to change it we should do both.

I prefer the "platform is fixed" idea where fast-usdc is independent of inter-protocol. But I can live with this.